Sources/MapboxNavigation/Resources/en.lproj/Localizable.strings

```diff
@@ -1,3 +1,9 @@
+/* Combined alternatives selection notes about duration and distance delta. */
```

Comment:
Document that the first placeholder is for the difference in time and the second placeholder is for the difference in distance.

Reply:
But it can be reverse and still remain correct.

Reply:
Yes, the localization can reverse the order of the placeholders in the translated string, but in order to do that, they need to know what values we set `%1$@` and `%2$@` to.